src/main/java/com/load/service/loadServiceImpl.java

```
@@ -0,0 +1,91 @@
+package com.load.service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import com.load.entities.Payload;
+import com.load.model.PayloadRequest;
+
+import com.load.dao.PayloadRepository;
+
+import com.load.model.PayloadResponse;
+
+
+@Service
+public class loadServiceImpl implements loadService {
+
+	@Autowired
+	PayloadRepository payloadRepository;
+	
+	//@Override
+//	public List<Payload> getAllPayloads() {
+//		return payloadRepository.findAll();
+//	}
+
+	
+	
+	@Override
+	public List<Payload> getPayloadById(Long payloadId) {
+		if(payloadId!=null)
+			 return payloadRepository.findAllById(payloadId);
+		return payloadRepository.findByStatus("pending");
+		
+	}
+
+	
+	public PayloadResponse createPayload(PayloadRequest payLoadRequest) {
+		// TODO Auto-generated method stub
+		
+		
+		PayloadResponse loadResponse = new PayloadResponse();
+		Payload load = new Payload();
+		
+		load.setShipperId(UUID.randomUUID());
+		if(payLoadRequest.getComment()!=null) {
+			load.setComment(payLoadRequest.getComment());
+		}
+		load.setUnloadingPoint(payLoadRequest.getUnloadingPoint());
+		load.setLoadingPoint(payLoadRequest.getLoadingPoint());
```

While assigning a value, you should check whether it is optional or not. if it is not optional you should return an error message if the value is not provided.